analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/LocksReleasedAllPaths.Mutex.NetFx.cs

```diff
@@ -94,7 +94,7 @@ public static void CompliantFromDocs(string mutexName, MutexSecurity mutexSecuri
             // Enter the mutex, and hold it until the program exits.
             try
             {
-                m.WaitOne();
+                m.WaitOne(); // Noncompliant FP
```

Comment:
In this test case, the states are different than before, and we see the FP in another place in the file. #6994 fixes the root cause for this flaky behavior.